src/app/views/provisioning/product-pricing-price.coffee

```diff
@@ -0,0 +1,8 @@
+@App.directive('mnoeProductPricingPrice', () ->
```

Comment:
Might want to rename this to make it plural as the component will display all of the prices, not just one.

Reply:
sure